assembl/graphql/synthesis.py

```diff
@@ -27,6 +27,7 @@ class Meta:
     ideas = graphene.List(lambda: IdeaUnion)
     img = graphene.Field(Document)
     creation_date = DateTime()
+    post = graphene.Field("assembl.graphql.post.Post")
```

Comment:
why not keep the name "published_in_post" for consistency and clarity?

Reply:
Why keep? It's `publishes_in_post` in the sqlalchemy relationship. We should maybe rename the relationship to `published_in_post`?

Reply:
Yeah, this was an old typo, prior to the era of CRs. Happy to see the relationship updated throughout the codebase. Not in scope of this PR, however.